parser.rb

```diff
@@ -6,13 +6,31 @@ module Parser
   def read(filepath)
     # binding.pry
     array = []
-    File.readlines(filepath, 'r').each do |line|
+    File.readlines(filepath).each do |line|
```

Comment:
@lksevans12  I'm not sure how File.readlines works, but this looks like you might be able to substitute with map, since you're instantiating a new array, adding to that array, and then returning that array.